### PR TITLE
Resolvendo issue-1

### PR DESCRIPTION
**Pull request by emmerson**
Issue 1

### CODE REVIEW

**Comment by eminetto** [comment]
Hello, @emmerson! This is your first Pull Request that will be reviewed by [Ebert](https://ebertapp.io/), an automatic Code Review service. It will leave comments on this diff with potential issues and style violations found in the code as you push new commits. You can also see all the issues found on this Pull Request on its [review page](https://ebertapp.io/github/Coderockr/exercicio-git/pulls/22). Please check our [documentation](https://ebertapp.io/docs) for more information.

File: tests/JokenPoTest.php [hunk 1]
```diff
@@ -26,4 +33,3 @@ public function testRules()
         $this->assertContains($rules, $jokenpo->rules());
     }
 }
-
```
Comment (jeanCarloMachado) [inline tests/JokenPoTest.php]: É importante deixar a última linha,

**Comment by emmerson** [comment]
O editor não está colaborando.

File: tests/JokenPoTest.php (hunk 1 above)
Comment (jeanCarloMachado) [inline tests/JokenPoTest.php]: Tem um espaço a mais nessa última linha.

**Pull request merged**